free-courses-en.md

```diff
@@ -153,6 +154,13 @@
 * [Intro to Parallel Programming Using CUDA to Harness the Power of GPUs](https://www.udacity.com/course/intro-to-parallel-programming--cs344) (Udacity)
 
 
+### Dart
+
+* [Dart Course for Beginners](https://www.udemy.com/course/dartlang) (Udemy)
+* [Dart Programming Tutorial | Learn the Dart for Flutter](https://www.udemy.com/course/dart-programming-tutorial-learn-the-dart-for-flutter) (Udemy)
+* [Dart Programming Tutorial - Full Course](https://www.youtube.com/watch?v=Ej_Pcr4uC2Q) - FreeCodeCamp
```

Comment:
Apparently, linter would like you to switch around those lines as well.

You can also view the build output yourself [here](https://travis-ci.com/github/EbookFoundation/free-programming-books/builds/191008848).